### PR TITLE
feat: add cmek support in composer_env_v2

### PR DESCRIPTION
**Pull request by Sarapuce**
Fixes https://github.com/terraform-google-modules/terraform-google-composer/issues/74

### CODE REVIEW

File: modules/create_environment_v2/main.tf [hunk 1]
```diff
@@ -199,6 +199,16 @@ resource "google_composer_environment" "composer_env" {
       }
     }
 
+    dynamic "encryption_config" {
+      for_each = var.kms_key_name != null ? [
```
Comment (imrannayer) [inline modules/create_environment_v2/main.tf:203]: @Sarapuce thanks for the PR. Can this be simplified as following 

```
for_each = var.kms_key_name != null ? ["encryption_config"] : []
      content {
        kms_key_name = var.kms_key_name
      }
}

```

Comment (Sarapuce) [reply modules/create_environment_v2/main.tf]: Done :)

**Comment by Sarapuce** [comment]
@imrannayer I took your comment in account, do I need to do something else ?

**Review by imrannayer — APPROVED** [review]

**Pull request merged**